zkfacade/src/main/java/com/yahoo/vespa/curator/stats/LockInfo.java

```diff
@@ -0,0 +1,99 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.curator.stats;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Optional;
+
+/**
+ * Information about a lock.
+ *
+ * <p>Should be mutated by a single thread, except {@link #fillStackTrace()} which can be
+ * invoked by any threads.  Other threads may see an inconsistent state of this instance.</p>
+ *
+ * @author hakon
+ */
+public class LockInfo {
```

Comment:
How about `LockAttempt`? `Info` is very generic.

Reply:
Done